src/bindings/jna/CMakeLists.txt

```diff
@@ -27,6 +27,21 @@ if (Java_VERSION_MAJOR GREATER 0 AND Java_VERSION_MINOR GREATER 7)
 		NAMES jna
 		PATHS /usr/share/java
 		)
+	
+	if(NOT jna) # try fetching it, remember to update SHA1 if you update the jar
+		file(DOWNLOAD 
+			"https://maven.java.net/content/repositories/releases/net/java/dev/jna/jna/4.2.2/jna-4.2.2.jar" 
```

Comment:
Isn't the jar bundled with elektra later such that the original jna.jar can be thrown away?
Otherwise yes, I can move it somewhere. I was trying not to magically move files into system directories just for `make` to work.